src/style.css

```diff
@@ -1 +1,62 @@
-/* add styles here */
+@font-face {
+  font-family: Avenir;
+  src: url('./fonts/Avenir-Book.ttf');
+  src: url('./fonts/Avenir-Heavy.ttf');
+  src: url('./fonts/Avenir.ttc');
```

Comment:
Having multiple `src` properties in the `@font-face` rule is not correct. You should combine them into a single `src` property using a comma to separate different font files. This ensures that the browser can choose the best format it supports.